src/utils/telemetry.js

```diff
@@ -18,54 +9,24 @@ const packageJson = JSON.parse(fs.readFileSync(new URL('../../package.json', imp
  * @template F
  * @param {{enabled: boolean}} options
  * @param {() => F | Promise<F>} callback
- * @returns {Promise<F>}
+ * @returns {Promise<void>}
  */
 export async function withTelemetry(options, callback) {
-  const { sentryHub, sentryClient } = createSentryInstance(options.enabled);
-
-  makeMain(sentryHub);
-
-  const transaction = sentryHub.startTransaction({
-    name: 'sentry-migr8-execution',
-    status: 'ok',
-    op: 'migr8.flow',
-  });
-  sentryHub.getScope().setSpan(transaction);
-  const sentrySession = sentryHub.startSession();
-  sentryHub.captureSession();
-
-  try {
-    return await runWithAsyncContext(() => callback());
-  } catch (e) {
-    sentryHub.captureException('Error during migr8 execution.');
-    transaction.setStatus('internal_error');
-    sentrySession.status = 'crashed';
-    throw e;
-  } finally {
-    transaction.finish();
-    sentryHub.endSession();
-    await sentryClient.flush(3000);
-  }
-}
-
-/**
- * Creates a minimal Sentry instance
- * @param {boolean} enabled
- * @returns {{sentryHub: Hub, sentryClient: NodeClient}}
- */
-function createSentryInstance(enabled) {
-  const client = new NodeClient({
+  Sentry.init({
```

Comment:
what we had before worked but I think for migr8 it's fine to actually simply call Sentry.init. We don't have the same leaking concerns as in the plugins. Additionally, we still don't add global integrations